app/src/main/java/org/stepik/android/view/step_quiz_choice/ui/adapter/ChoicesAdapterDelegate.kt

```diff
@@ -53,6 +54,7 @@ class ChoicesAdapterDelegate(
 
             itemChoiceFeedback.setTextSize(14f)
             itemChoiceFeedback.setBackgroundResource(R.drawable.bg_step_quiz_choice_item_feedback)
+            itemChoiceLatex.findViewById<LatexSupportableEnhancedFrameLayout>(R.id.latex_text).setTextIsSelectable(true)
```

Comment:
It's better to store reference for this view instead of executing findViewById every time